src/Command/Setup.php

```diff
@@ -81,7 +84,10 @@ protected function execute(InputInterface $input, OutputInterface $output)
     public function validateVar($value)
```

Comment:
IMHO, this method does validate vars, indeed, but is specific to the `app` & `vendor` vars. Of course those are the only vars now, but this method thus isn't a generic `validateVar` one for further development, or am I wrong ? :/

Above all, I think it should be moved into a dedicated class, but I failed to find a good candidate. WDYT ? :)